src/pages/DatasetStatistics.jsx

```diff
@@ -78,52 +78,28 @@ export default function DatasetStatistics(props) {
               <div style={{ ...Styles.MINOR_HEADER, paddingLeft: '10px' }}>Dataset Description:</div>
               {LINE}
               <div style={{ fontSize: Theme.font.size.small, padding: '1rem' }}>
-                {get(
-                  find(dataset?.properties, (p) => {
-                    return p.propertyName === 'Description';
-                  }),
-                  'propertyValue',
-                  ''
-                )}
+                {extract('Dataset Description') || dataset?.description || dataset?.study?.description }
               </div>
             </div>
             <div>
               <div style={{ display: 'flex' }}>
                 <div style={Styles.SMALL_BOLD}>Number of Participants: </div>
                 <div style={Styles.SMALL_BOLD}>
-                  {get(
-                    find(dataset?.properties, (p) => {
-                      return p.propertyName === '# of participants';
-                    }),
-                    'propertyValue',
-                    ''
-                  )}
+                  {extract('# of participants')}
                 </div>
               </div>
-              <div style={{ display: 'flex' }}>
+              {(extract('Principal Investigator(PI)') || dataset?.study?.piName) && <div style={{ display: 'flex' }}>
                 <div style={Styles.SMALL_BOLD}>Principal Investigator: </div>
                 <div style={Styles.SMALL_BOLD}>
-                  {dataset?.study?.piName || get(
-                    find(dataset?.properties, (p) => {
-                      return p.propertyName === 'Principal Investigator(PI)';
-                    }),
-                    'propertyValue',
-                    ''
-                  )}
+                  {extract('Principal Investigator(PI)') || dataset?.study?.piName}
```

Comment:
One of the things I realized in this code is that the `dataset` object isn't a real dataset as we understand it, but instead is a `DatasetDTO` [object](https://github.com/DataBiosphere/consent/blob/develop/src/main/java/org/broadinstitute/consent/http/models/DatasetMetrics.java#L8) which is a deprecated class. We'll definitely need some back end work to make this more accurate.

Reply:
Yes, this is why I extract the dataset directly instead using `DataSet.getDataSetsByDatasetId`